cmds/core/chmod/chmod.go

```diff
@@ -191,7 +211,7 @@ func main() {
 		reference = flag.String("reference", "", "use mode from reference file")
 	)
 	flag.Parse()
-	if _, err := chmod(*recursive, *reference, flag.Args()...); err != nil {
-		log.Fatal(err)
+	if err := command(os.Stderr, *recursive, *reference).run(flag.Args()...); err != nil {
+		os.Exit(1)
```

Comment:
if you did use errors.Join, above, this change might not be needed?

Reply:
Even with error.Join if we want to the same behaviour as regular chmod we need to change log.Fatal to something like fmt.Sprintf(os.Stderr, errors) and that call os.Exit(1)

Reply:
somehow my comment on this got lost. 
https://go.dev/play/p/UMgj4neXlI6
so you can, given something constructed with errors.Join
log.Fatal(err)
and that works.

Reply:
ok, lets do it with errors.Join, looks like CI is already moved beyond 1.19

Reply:
I think no error.Join until February :)

# github.com/u-root/u-root/cmds/core/chmod
core/chmod/chmod.go:194:19: undefined: errors.Join
core/chmod/chmod.go:199:19: undefined: errors.Join
note: module requires Go 1.21

Exited with code exit status 2

So probably it make sense to merge this as is and cleanup a bit after we abandon 1.19

Reply:
arg, so annoying. you are right.